phpBB/develop/adjust_avatars.php

```diff
@@ -30,6 +30,11 @@
 	die('database not up to date');
 }
 
+if (!isset($config['storage\\avatar\\config\\path']) || $config['storage\\avatar\\config\\path'] !== 'phpbb\\storage\\provider\\local')
+{
+	die('use local provider');
```

Comment:
I think this message should be improved as for someone not knowing the exact purpose of this might not be able to understand why it's causing a die().

Reply:
I think i can rewrite this to use storage, but i dont know under which circunstances this script is used, so i just prevent to use it after changing storage

Reply:
Actually I think we can leave it like that for now. We should start working on the dev doc soon and include a link here